src/controllers/usersController.js

```diff
@@ -13,16 +13,15 @@ const usersIndex = (req, res) => {
       const sessionID = data.body.id
       Users.find({}).lean()
         .then((result) => {
-          if (result === undefined) {
+          if (!result) {
             res.render('home', {
               heartIcon,
               banner: mainBanner
             })
           } else {
             const myProfile = result.find((profile) => profile.id.includes(sessionID))
-            console.log(myProfile)
             const userProfiles = result.filter(
-              (user) => !myProfile.likes.includes(user.id) && !myProfile.dislikes.includes(user.id)
+              (user) => !myProfile?.likes.includes(user.id) && !myProfile?.dislikes.includes(user.id)
```

Comment:
waar is dat vraagteken voor?

Reply:
Ik kwam er niet helemaal uit en was dus even aan het bellen met Victor Boucher (student assistent). Er ontstond hier een error, ik geloof dat het op een manier niet defined was. Het vraagteken lost dit op. Ik moet je eerlijk zeggen dat ik even kwijt ben wat het precies doet maar dat heb ik hem net dus ook al nog een keer gevraagd. dus ik kom hier nog op terug :)

Reply:
Oke, als myprofile undefined is zou je een error krijgen omdat likes niet bestaat op undefined. En met het vraagteken check je eigenlijk eerst of myprofile bestaat en als ie bestaat pakt ie de likes etc. Je voorkomt dus de error als myprofile er niet is, of geen waarde heeft.

Reply:
aah yes! snap hem, moet daar ook nog even naar kijken, hij geeft inderdaad undefined omdat hij in de eerste keer al dingen van een profiel probeert op te halen ook als het profiel nog niet gemaakt is, maar geloof dat het zo al opgelost is, lekker bezig man!